.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ develop ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: SSH action
+        uses: appleboy/ssh-action@v0.1.3
+        with:
+          HOST: ${{ secrets.SSH_HOST }}
```

Comment:
i don't see secrets matching these names in github; can you make sure to add so this can be tested?

Reply:
Yes we will need these set by the user who has ssh access. probably @Ryan-Koch

Reply:
i assume we'll have separate secret values for staging and production that we'll eventually want to use with this workflow.  i created a new Environment called staging (access via the Settings of this repo) and added all three of these keys.  let me know what else you need to get this tested.